pay-api/src/pay_api/models/fee_schedule.py

```diff
@@ -91,6 +92,16 @@ def find_all(cls, corp_type_code: str = None, filing_type_code: str = None):
         if corp_type_code:
             query = query.filter_by(corp_type_code=corp_type_code)
 
+        if description:
+            # description is free text search
+            description_list = description.replace(' ', '%')
+            query = query.join(CorpType,
+                               CorpType.code == FeeSchedule.corp_type_code). \
+                join(FilingType, FilingType.code == FeeSchedule.filing_type_code)
+            query = query.filter(
+                or_(func.lower(FilingType.description).contains(description_list.lower(), autoescape=True),
```

Comment:
Shouldn't this be using `like` rather than `contains` ? with autoescape can `contains` handle multiple `%` ?

Reply:
hm..yeah..i am planning to rewrite this...  i was trying to split the string and do a like in both of the fields ; i think thats the way to go...the challenge with multiple % is basically we are searching same string in two columns ...might not be present in one column ;but in other... so have to figure it out...   I will definitely rewrite it after the core functionality of payment is done..